<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Endstops_Configurator</class>
 <widget class="QDialog" name="Endstops_Configurator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>403</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>403</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>403</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Endstop Configuration </string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>://iloop_icon_small.png</normaloff>://iloop_icon_small.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
        background: rgb(33, 37, 43);
        color: white;
        font-family: &quot;Source Sans Pro&quot;;
}

QCheckBox {
        padding: 5px;
        border: 1px solid transparent;
        border-radius: 3px;
}

QCheckBox::hover {
        border: 1px solid rgb(54, 60, 70);
}

QCheckBox::indicator {
    width: 15px;
    height: 15px;
        background: rgb(54, 60, 70);
}

QCheckBox::indicator:checked {
        image: url(&quot;://check_FILL0_wght400_GRAD0_opsz20.svg&quot;);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <item row="13" column="0">
    <widget class="QFrame" name="frame_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame {
        background: rgb(38, 43, 50);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Close</set>
        </property>
        <property name="centerButtons">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QFrame" name="frame_4">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="0">
       <widget class="QTabWidget" name="tabWidget">
        <property name="styleSheet">
         <string notr="true">QTabWidget::pane {
        border: none;
}

QTabBar::tab {
        background: rgb(38, 43, 50);
        border: 1px solid rgb(54, 60, 70);
        border-bottom: none;
        padding: 5px 10px;
        border-top-left-radius: 3px;
    border-top-right-radius: 3px;
        margin: 0 1px;
}

QTabBar::tab:!selected {
    margin-top: 2px;
}

QTabBar::tab:selected {
    background: rgb(54, 60, 70);
}</string>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Available Endstops</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="verticalSpacing">
           <number>10</number>
          </property>
          <item row="0" column="0" rowspan="2" colspan="2">
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="4" column="0">
              <widget class="QCheckBox" name="uUseIminPlug">
               <property name="text">
                <string>USE_IMIN_PLUG</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QCheckBox" name="uUseXminPlug">
               <property name="text">
                <string>USE_XMIN_PLUG</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="7" column="0" colspan="2">
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="6" column="0">
              <widget class="QCheckBox" name="uUseKminPlug">
               <property name="text">
                <string>USE_KMIN_PLUG</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QCheckBox" name="uUseJmaxPlug">
               <property name="text">
                <string>USE_JMAX_PLUG</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QCheckBox" name="uUseYmaxPlug">
               <property name="text">
                <string>USE_YMAX_PLUG</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QCheckBox" name="uUseJminPlug">
               <property name="text">
                <string>USE_JMIN_PLUG</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QCheckBox" name="uUseImaxPlug">
               <property name="text">
                <string>USE_IMAX_PLUG</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0" colspan="2">
              <widget class="QLabel" name="label">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="text">
                <string>Which limit switches are available in the printer?</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QCheckBox" name="uUseXmaxPlug">
               <property name="text">
                <string>USE_XMAX_PLUG</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QCheckBox" name="uUseZmaxPlug">
               <property name="text">
                <string>USE_ZMAX_PLUG</string>
               </property>
              </widget>
             </item>
             <item row="6" column="1">
              <widget class="QCheckBox" name="uUseKmaxPlug">
               <property name="text">
                <string>USE_KMAX_PLUG</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QCheckBox" name="uUseZminPlug">
               <property name="text">
                <string>USE_ZMIN_PLUG</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QCheckBox" name="uUseYminPlug">
               <property name="text">
                <string>USE_YMIN_PLUG</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Invert Endstops</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="verticalSpacing">
           <number>10</number>
          </property>
          <item row="0" column="0" colspan="2">
           <widget class="QFrame" name="frame_2">
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="1" column="1">
              <widget class="QCheckBox" name="uXmaxEndstopInverting">
               <property name="text">
                <string>X_MAX_ENDSTOP_INVERTING</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QCheckBox" name="uXminEndstopInverting">
               <property name="text">
                <string>X_MIN_ENDSTOP_INVERTING</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QCheckBox" name="uIminEndstopInverting">
               <property name="text">
                <string>I_MIN_ENDSTOP_INVERTING</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QCheckBox" name="uYmaxEndstopInverting">
               <property name="text">
                <string>Y_MAX_ENDSTOP_INVERTING</string>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QCheckBox" name="uKminEndstopInverting">
               <property name="text">
                <string>K_MIN_ENDSTOP_INVERTING</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QCheckBox" name="uImaxEndstopInverting">
               <property name="text">
                <string>I_MAX_ENDSTOP_INVERTING</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QCheckBox" name="uJminEndstopInverting">
               <property name="text">
                <string>J_MIN_ENDSTOP_INVERTING</string>
               </property>
              </widget>
             </item>
             <item row="6" column="1">
              <widget class="QCheckBox" name="uKmaxEndstopInverting">
               <property name="text">
                <string>K_MAX_ENDSTOP_INVERTING</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QCheckBox" name="uZmaxEndstopInverting">
               <property name="text">
                <string>Z_MAX_ENDSTOP_INVERTING</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0" colspan="2">
              <widget class="QLabel" name="label_2">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="text">
                <string>Which limit switch states should be inverted?</string>
               </property>
              </widget>
             </item>
             <item row="7" column="0">
              <widget class="QCheckBox" name="uZminProbeEndstopInverting">
               <property name="text">
                <string>Z_MIN_PROBE_ENDSTOP_INVERTING</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QCheckBox" name="uZminEndstopInverting">
               <property name="text">
                <string>Z_MIN_ENDSTOP_INVERTING</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QCheckBox" name="uYminEndstopInverting">
               <property name="text">
                <string>Y_MIN_ENDSTOP_INVERTING</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QCheckBox" name="uJmaxEndstopInverting">
               <property name="text">
                <string>J_MAX_ENDSTOP_INVERTING</string>
               </property>
              </widget>
             </item>
             <item row="8" column="0" colspan="2">
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Endstops_Configurator</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Endstops_Configurator</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
